apps/idos-data-dashboard/src/core/wagmi.ts

```diff
@@ -13,7 +13,7 @@ const metadata = {
   icons: ["/idos-dashboard-logo.svg"]
 };
 
-export const chains = [mainnet];
+export const chains = [import.meta.env.DEV ? sepolia : mainnet];
```

Comment:
For local environment with using local Hardhat chain it is `localhost` needed. For `Sepolia` usage it is ok to use mainnet (it works on `staging` well).
```suggestion
export const chains = [import.meta.env.DEV ? localhost : mainnet];
```